### PR TITLE
Docs/fix item links

### PR DESCRIPTION
**Pull request by jost-s**
### Summary
in crates `holochain_conductor_api` and `holochain_state`

### TODO:
- [x] CHANGELOG(s) updated with appropriate info
- [x] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

File: crates/holochain_state/src/lib.rs [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 //! # Persisted State building blocks
 //!
-//! This crate provides a few types for working with databases. The types build upon those found in [holochain_sqlite::buffer].
-//!
-//! - [ElementBuf]: the union of two CasBuffers, one for Entries, one for Headers
-//! - [ChainSequenceBuf]: database representing the chain sequence DB, which provides a special method for accessing the chain head
-//! - [SourceChainBuf]: the union of a [ElementBuf] and a [ChainSequenceBuf], which fully represents a source chain
-//! - [MetadataBuf]: (*unimplemented*) Uses a KvvBuffer to represent EAV-like relationships between CAS entries
-//! - [Cascade]: (*unimplemented*) Unifies two [ElementBuf] and two [MetadataBuf] references (one of each is a cache) in order to perform the complex metadata-aware queries for getting entries and links, including CRUD resolution
+//! This crate provides a few types for working with databases. The types build upon those found in holochain_sqlite::buffer.
```
Comment (freesig) [inline crates/holochain_state/src/lib.rs:4]: These docs are really out of date. I don't think the buf types even exist anymore?

**Review by freesig — APPROVED** [review]
Looks good except for the buf types.

File: crates/holochain_state/src/lib.rs (hunk 1 above)
Comment (jost-s) [reply crates/holochain_state/src/lib.rs:4]: @freesig That's right, Michael said that too. I've added a `todo` to the docs for a future update.

File: crates/holochain_p2p/src/types/actor.rs [hunk 2]
```diff
@@ -18,45 +18,45 @@ pub struct GetValidationPackage {
...
 #[derive(Clone, Debug)]
 /// Get options help control how the get is processed at various levels.
-/// Fields tagged with `[Network]` are network-level controls.
-/// Fields tagged with `[Remote]` are controls that will be forwarded to the
+/// Fields tagged with ```[Network]``` are network-level controls.
+/// Fields tagged with ```[Remote]``` are controls that will be forwarded to the
 /// remote agent processing this `Get` request.
 pub struct GetOptions {
-    /// [Network]
+    /// ```[Network]```
```
Comment (maackle) [inline crates/holochain_p2p/src/types/actor.rs:25]: What do triple ticks mean?

File: crates/holochain_state/src/lib.rs (hunk 1 above)
Comment (maackle) [reply crates/holochain_state/src/lib.rs:4]: These are so out of date that having no docs would be better than any here. It's a totally different paradigm. We won't even have such a diagram in the future.

**Review by maackle — APPROVED** [review]
Looks good, thanks for doing this.

Will we have something in CI now to help keep the links from deteriorating again?

File: crates/holochain_p2p/src/types/actor.rs (hunk 2 above)
Comment (jost-s) [reply crates/holochain_p2p/src/types/actor.rs:25]: When I use single ticks with the brackets, rustdoc interprets it as an intra-doc link. With triple ticks it'll be printed as `code` and with brackets, but not seen as intra-doc link.

**Comment by jost-s** [comment]
> Will we have something in CI now to help keep the links from deteriorating again?

Yes, I've added a job.

**Pull request merged**